Week3/homework/step2-2.js

```diff
@@ -2,19 +2,28 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
-
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+    if (i % 3 === 0 && i % 5 === 0) {
+      threeCallback(i);
+      fiveCallback(i);
+    }
+    if (i % 3 === 0 && i % 5 !== 0) {
+      threeCallback(i);
+    } else if (i % 5 === 0 && i % 3 !== 0) {
+      fiveCallback(i);
+    }
+  }
```

Comment:
Hi Yash, thanks for the feedback. As you recognized, we have a small group :) first, each one tries to complete hw alone. Then, we discuss a problem, which we get stuck on, to find a solution.

Reply:
Ah okay. I understand. But could you please let the reviewer know the next time you work together with someone on a homework assignment? It would be easier for that person while reviewing. 🙂